modules/persistence/index.ts

```diff
@@ -55,15 +55,7 @@ const app = async (path: string, githubUser: string) => {
   }
 };
 
-try {
-  console.log(argv);
-  app(argv['path'], argv['githubUser']);
-} catch (error) {
-  console.log('caught in terminal handling');
-  // only exit with non zero error code if running with strict mode on
-  if (['y', 'yes', 'true'].includes(argv['strict'].toLowerCase())) {
-    process.exit(1);
-  } else {
-    process.exit(0);
-  }
-}
+app(argv['path'], argv['githubUser']).catch(() => {
+  console.error('Persistence Module Failure. Ending build.');
```

Comment:
I noticed that you're not printing the error that the catch catches. for example:
```
.catch((err) = > 
      console.error('Persistence Module failure, ending build:', err)
```
However I don't know we need to print this, maybe its redundant so please feel free to ignore this comment.

Reply:
Great callout. We talked in person, but putting here for visibility. I did originally log out the error here, but I realized all of the thrown errors throughout the module already log out the error before throwing, so it resulted in a double log.